tutorials/clustering.md

```diff
@@ -63,37 +63,54 @@ using our informational views:
 SELECT * FROM timescaledb_information.data_node;
 ```
 
-Now that we've created a database and connected a couple data nodes, let's go ahead
-and create a distributed hypertable. With the following commands, we'll create
-a hypertable and then distribute it:
+Now that we've created a database and connected a couple data nodes,
```

Comment:
Suggestion: s/connected/added/ This is consistent with the name of `add_data_node`.